### PR TITLE
added crash handler

### PR DESCRIPTION
**Pull request by localcc**
This crash handler allows for crashdump generation.

### CODE REVIEW

File: assets/UE4SS-settings.ini [hunk 1]
```diff
@@ -87,6 +87,7 @@ MakeAllConfigsEngineConfig = 1
 ConsoleEnabled = 1
 GuiConsoleEnabled = 1
 GuiConsoleVisible = 1
+FullMemoryDump = 0
```
Comment (narknon) [inline assets/UE4SS-settings.ini:90]: I don't really like this being grouped.  At this point the live view debugger is more of a mod debugger than a UE4SS debugger, and I think the 2 concepts should be distinguished.

**Comment by narknon** [comment]
Please add an option to disable it completely (but maybe move the settings all the way to the bottom above experimental features).

**Comment by UE4SS** [comment]
> Please add an option to disable it completely (but maybe move the settings all the way to the bottom above experimental features).

What is the purpose of this ?

**Comment by narknon** [comment]
Would you want a program that dumped 20mb of data every time it crashed with absolutely no option to turn it off? Hell, I don't want it on when I'm testing stuff I know will probably crash 50 times.

**Comment by UE4SS** [comment]
> Would you want a program that dumped 20mb of data every time it crashed with absolutely no option to turn it off? Hell, I don't want it on when I'm testing stuff I know will probably crash 50 times.

Personally, wouldn't affect me, I'm always gonna have it enabled, but I see your point.

**Comment by narknon** [comment]
Tfw you forget your c drive is almost full and end up zip bombed by a crash dump (UE has done this to me several times)

**Pull request merged**